resources/templates/course.html

```diff
@@ -0,0 +1,39 @@
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="{{cssPath}}">
+        <link rel="stylesheet" type="text/css" href="{{bootstrapPath}}">
+
+        <script>
+            const vscode = acquireVsCodeApi();
+            function sendCourse(id) {
+                vscode.postMessage({ type: "setCourse", id });
+            }
+        </script>
+    </head>
+
+    <body>
+        <div class="container container-fluid">
+            <div class="orgHeader row">
+                <div class="col-md-2">
+                    <img class="img-fluid orgImg" src="{{resolve_logo_path organization.logo_path}}" />
```

Comment:
There is some inconsistency with how these classes are named here. Is this a bootsrap/own class separation?

Reply:
Will be fixed today.